src/modules/FSXARemoteApi.ts

```diff
@@ -464,6 +458,12 @@ export class FSXARemoteApi implements FSXAApi {
         case 401:
           throw new Error(FSXAApiErrors.NOT_AUTHORIZED)
         default:
+          if (response.status === 400) {
```

Comment:
To be able to handle this error in our Nuxt stack (FSXAProxyApi) we should probably throw a specific Error for general client errors (400 responses) and include the response body. We can then catch and include it in the response for the original request of the FSXAProxyApi.

Reply:
We decided to skip this.